_posts/00-01-00-Before-You-Begin.md

```diff
@@ -0,0 +1,45 @@
+---
+anchor: important_coding_tip
+---
+
+# Important Coding Tip{#important_coding_tip_title}
+
+## Do not do as they do
+As you learn how to program in PHP you will find almost all instructional tutorials as of 2014 do you a grave injustice.  They teach you how to write dangerous, hackable, insecure PHP code.
```

Comment:
"many" might be more acceptable. There's a lot of stuff out there that you'll be insulting which isn't that bad.

I've written plenty of tutorials that still stand, as have some of my friends. :)

--
Phil Sturgeon
Sent from my iPhone and there's probably typos because I'm probably at the pub.

> On May 31, 2015, at 4:18 PM, Scott Arciszewski <notifications@github.com> wrote:
> 
> In _posts/00-01-00-Before-You-Begin.md:
> 
> > @@ -0,0 +1,45 @@
> > +---
> > +anchor: important_coding_tip
> > +---
> > +
> > +# Important Coding Tip{#important_coding_tip_title}
> > +
> > +## Do not do as they do
> > +As you learn how to program in PHP you will find almost all instructional tutorials as of 2014 do you a grave injustice.  They teach you how to write dangerous, hackable, insecure PHP code.
> Actually, "almost all" reflects my experience.
> 
> —
> Reply to this email directly or view it on GitHub.
>